plugins/parcelport/mpi/CMakeLists.txt

```diff
@@ -47,8 +47,8 @@ if(HPX_WITH_PARCELPORT_MPI)
     endif()
 
     set(_mpi_libraries)
-    if(MPI_CXX_LIBRARIES)
-      set(_mpi_libraries ${_mpi_libraries} ${MPI_CXX_LIBRARIES})
+    if(MPI_C_LIBRARIES)
```

Comment:
It may be the case that with some older cmake versions that you had problems, however - this is the correct behaviour and CXX libs should not be needed. I suggest you give this a try and if you run into problems on some machines then I will revert this.